sdk/react/panels/style/panelStyles.json

```diff
@@ -37,29 +35,29 @@
     "fontSize": 20,
     "fontFamily": "Helvetica",
     "fontWeight": "bold",
-    "color": "white",
+    "color": "white"
   },
   "closedCaptionsContainer": {
-    "backgroundColor":"transparent",
-    "alignItems":"center",
-    "justifyContent":"center",
+    "backgroundColor": "transparent",
+    "alignItems": "center",
+    "justifyContent": "center"
   },
   "closedCaptions": {
-    "alignSelf":"center",
-    "textAlign":"center",
+    "alignSelf": "center",
+    "textAlign": "center",
     "padding": 2,
-    "margin":2,
+    "margin": 2,
     "fontWeight": "bold",
     "fontSize": 16,
     "fontFamily": "Helvetica",
     "color": "white",
-    "backgroundColor": "rgba(0, 0, 0, 0.7)",
+    "backgroundColor": "rgba(0, 0, 0, 0.7)"
   },
   "closedCaptionsFlexibleSpace" : {
```

Comment:
remove space between `"closedCaptionsFlexibleSpace"` and `:`

Reply:
fixed